src/main/tray.js

```diff
@@ -0,0 +1,163 @@
+/*!
+GPII Application
+Copyright 2016 Steven Githens
+Copyright 2016-2017 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+"use strict";
+
+var fluid    = require("infusion");
+var path     = require("path");
+var electron = require("electron");
+
+var Tray           = electron.Tray,
+    globalShortcut = electron.globalShortcut;
+var gpii           = fluid.registerNamespace("gpii");
+
+/**
+ * Component that contains an Electron Tray.
+ */
+fluid.defaults("gpii.app.tray", {
+    gradeNames: "fluid.modelComponent",
+    members: {
+        tray: {
+            expander: {
+                funcName: "gpii.app.makeTray",
+                args: ["{that}.options.icons.keyedOut", "{psp}.show"]
+            }
+        }
+    },
+    icons: {
+        pendingChanges: "../icons/gpii-pending.png",
```

Comment:
Could you update these references to support and use package-relative paths such as "%gpii-app/icons/gpii-pending.png" - this will make the work easier in future of migrating such assets into their own module as has been requested. You can achieve this by passing such paths to https://docs.fluidproject.org/infusion/development/NodeAPI.html#fluidmoduleresolvepathpath in order to get a full filesystem path